aws-codeartifact-repository/aws-codeartifact-repository.json

```diff
@@ -37,6 +37,26 @@
       "type": "string",
       "minLength": 1,
       "maxLength": 2048
+    },
+    "ExternalConnections": {
+      "description": "An array of external connections associated with the repository.",
```

Comment:
Is it described anywhere that this only currently supports a single element?

Reply:
It's not anywhere in the resource handlers, I could add it in this description.